### PR TITLE
Implement PhysX Trigger Shapes.

### PR DESCRIPTION
**Pull request by gonnavis**
Fix: https://github.com/webaverse/app/issues/3183

Sibling PR: https://github.com/webaverse/app-wasm/pull/64

Related Repos:
https://github.com/webaverse/triggercube
https://github.com/webaverse/physicscube

Usage: https://github.com/webaverse/triggercube/blob/main/index.js#L110-L124

https://user-images.githubusercontent.com/10785634/177046741-5b4078b2-1a44-4ea1-be80-1c353a1058f3.mp4

image

### CODE REVIEW

**Comment by gonnavis** [comment]
App dispatch `triggerin` & `triggerout` events.

https://user-images.githubusercontent.com/10785634/177143313-caf5ea13-968c-4a39-92aa-dc19e35ee1db.mp4

**Comment by alisaad673** [comment]
Seems to be working fine.

https://user-images.githubusercontent.com/35926530/177357409-12a9da80-746f-49b4-9ebc-87be7f73e067.mp4

File: physics-manager.js [hunk 1]
```diff
@@ -754,6 +757,36 @@ class PhysicsScene extends EventTarget {
   getGravity() {
     return gravity;
   }
+  setTrigger(id) {
+    return physx.physxWorker.setTriggerPhysics(
+      this.scene, id,
+    )
+  }
+  getTriggerEvent() {
```
Comment (avaer) [inline physics-manager.js:765]: It's more than one event, so should be called `getTriggerEvent`.

**Review by avaer — APPROVED** [review]
One rename request, the rest looks good.

**Comment by gonnavis** [comment]
Renamed.
Sibling PR: https://github.com/webaverse/app-wasm/pull/65

**Pull request merged**